contributing-guides/translation-templates/common-arguments.md

```diff
@@ -21,7 +21,7 @@ Only the left-alignment of the header gets lost and has to be re-added again (`|
 | it    | percorso/al/file     | percorso/al/directory  | percorso/al/file_o_directory      | pacchetto |                   |
 | ja    |                      |                        |                                   |           |                   |
 | ko    |                      |                        |                                   |           |                   |
-| ml    |                      |                        |                                   |           |                   |
+| ml    |പാത/ഫയൽ              |പാത/ഡയറക്ടറി               | പാത/ഫയൽ_അല്ലെങ്കിൽ_ഡയറക്ടറി            |പാക്കേജ്      |ഉപയോക്തൃനാമം         |
```

Comment:
```suggestion
| ml    |പാത്ത്/ടു/ഫയൽ              |പാത/ടു/ഡയറക്‌ടറി             | പാത്ത്/ടു/ഫയൽ_ഓർ_ഡയറക്‌ടറി           |പാക്കേജ്      |ഉപയോക്തൃനാമം         |
```
@sanbabyfrancis @akash1729 Would this be better?
> Note: I translated this using Google Translate to add `to/` to paths, feel free to make changes.

Reply:
Yeah it would make sense to keep it പാത്ത്/ടു since the other one is kind of ambiguous. I have changed it

Reply:
1. `പാത/` is a native way of representing `path/to/` in Malayalam. 
2. `പാത്ത്/ടു` is simply the Malayalam translation of the words: `path` and `to`.

Both sound right in our context. If we were to strictly follow the style: `path/to/`, let's go with the second one.

Reply:
I am fine with anything. @navarroaxel What do you think?

Reply:
Also, in the above edit, replace `പാത/ടു/ഡയറക്‌ടറി` with `പാത്ത്/ടു/ഡയറക്‌ടറി` @kbdharun

Reply:
@kbdharun @akash1729 What do you guys think about `ഫയലിലേക്കുള്ള/പാത`, `ഡയറക്ടറിയിലേക്കുള്ള/പാത` and `ഫയലിലേക്കോ_ഡയറക്ടറിയിലേക്കോ/ഉള്ള/പാത`? I think we should translate the strings where possible rather than transliterate.

Reply:
This looks perfect! It is precise, grammatically structured and more readable than the transliterated string. Maybe we should adopt this instead of compromising the readability of the text by sticking to the traditional transliterated `path/to/directory` string.

Reply:
> @kbdharun @akash1729 What do you guys think about `ഫയലിലേക്കുള്ള/പാത`, `ഡയറക്ടറിയിലേക്കുള്ള/പാത` and `ഫയലിലേക്കോ_ഡയറക്ടറിയിലേക്കോ/ഉള്ള/പാത`? I think we should translate the strings where possible rather than transliterate.

Thanks, I will update my original review.

Reply:
Yeah this is perfect. This can use this approach.
Should we use `ഫയലിലേക്ക്/ഉള്ള/പാത` or `ഫയലിലേക്കുള്ള/പാത` ?
What do you guys think ?

Reply:
> Yeah this is perfect. This can use this approach. Should we use `ഫയലിലേക്ക്/ഉള്ള/പാത` or `ഫയലിലേക്കുള്ള/പാത` ? What do you guys think ?

I think we can go with this ` ഫയലിലേക്കുള്ള/പാത` as discussed above. You can commit the suggestion I made above.